tasks/pipeline-osdeps-MCPClient.yml

```diff
@@ -27,6 +27,13 @@
       when:
         - "MCPClient.osdeps_packages"
 
+    - name: "Install MCPClient ext. package deb file deps."
+      apt:
+        deb: "{{ item }}"
+      with_items: "{{ MCPClient.osdeps_debfiles }}"
+      when:
+        - "MCPClient.osdeps_debfiles"
+
```

Comment:
Does the role work fine if the `osdeps_debfiles` key is not present in the json file? In the CentOS/RH block below there are some tasks with `when` clauses that check if the key is defined (e.g.,  `MCPClient.osdeps_packages is defined and MCPClient.osdeps_packages`), can't remember if we did this because the playbook threw an error when this was not defined in the json file.